recipes-ccsp/ccsp/rdkxdslmanager.bb

```diff
@@ -0,0 +1,87 @@
+SUMMARY = "RdkXdslManager component"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
+
+DEPENDS = "ccsp-common-library dbus rdk-logger utopia json-hal-lib avro-c hal-platform libparodus libunpriv"
+require ccsp_common.inc
```

Comment:
Please change this to
`require recipes-ccsp/ccsp/ccsp_common.inc`
to resolve recipe parse error.

Reply:
Addressed. Can you pls review?